client/api/eventsApi.js

```diff
@@ -12,23 +19,38 @@ export function addNewEvent (id, event) {
     .catch((error) => console.log(error))
 }
 
+/**
+ * Request array of events from projects table
+ * @returns {array} All events
+ */
 export function showAllEvents () {
   return request
     .get('/api/v1/events')
     .set(acceptJsonHeader)
     .then((res) => res.body)
     .catch((error) => console.log(error))
 }
-
-export function joinEvent (id, eventID) {
+/**
+ * Add userID and EventId to users_projects table
+ * @param {string} id - User Id
+ * @param {string} eventId - eventId
+ *
+ * @returns {array} array[0] team id
+ */
+export function joinEvent (id, eventId) {
   return request
     .post('/api/v1/events/attending')
     .set(acceptJsonHeader)
-    .send(id, eventID)
+    .send(id, eventId)
     .then((res) => res.body)
     .catch((error) => console.log(error))
 }
-
+/**
+ * Get a list of all events user has joined
+ * @param {string} id - User Id
+ *
+ * @returns {array} A list of event ids
+ */
 export function showAllUserEvents (id) {
```

Comment:
💭 Just to be extra clear (because we have an existing function `showAllEvents()` that resolves with an array of event objects), it might be useful to rename this function to `showAllUserEventIds`, to be extra explicit that this function resolves with IDs, not events.

Reply:
Changed function name 👍, updated name in eventsApi.test as well